components/sections/ExportTranslations.tsx

```diff
@@ -75,17 +72,28 @@ const ExportTranslations = ({ translations, categoryName, triggerLabel }: Export
 
   const translSep = translationSeparator === TRANS_SEP_CUSTOM ? customTranslationSeparator : translationSeparator;
   const phraseSep = phraseSeparator === PHRASE_SEP_CUSTOM ? customPhraseSeparator : phraseSeparator;
-  const phrases = translations
-    .map(
-      (translation) =>
-        translation.getTranslation(currentLanguage) +
-        (includeTranscriptions ? ` [${translation.getTranscription(currentLanguage)}]` : '') +
-        translSep +
-        translation.getTranslation(otherLanguage) +
-        (includeTranscriptions ? ` [${translation.getTranscription(otherLanguage)}]` : '') +
-        phraseSep
-    )
-    .map((translation) => unescapeTabsAndNewlines(translation));
+  const source =
+    categoryName != 'všechny fráze'
+      ? translations.categories.filter((c) => `${c.name.main} - ${c.name.source}` === categoryName)
+      : translations.categories;
+  const phrases = source.map((translation) => {
+    const text = '[' + translation.name.main + ']' + translSep + '[' + translation.name.source + ']' + '\n';
```

Comment:
I think that translation.name.main and translation.name.source need to be switched when language is switched.